File: idscp2/src/main/java/de/fhg/aisec/ids/idscp2/error/Idscp2Exception.java
package de.fhg.aisec.ids.idscp2.error;

/**
 * IDSCP2 Exception
 *
 * @author Leon Beckmann (leon.beckmann@aisec.fraunhofer.de)
 */
public class Idscp2Exception extends RuntimeException {
    public Idscp2Exception(String message) {
        super(message);
    }

    public Idscp2Exception(String message, Throwable cause) {
        super(message, cause);
    }
}

File: idscp2/src/main/kotlin/de/fhg/aisec/ids/idscp2/error/Idscp2Exception.kt
package de.fhg.aisec.ids.idscp2.error

/**
 * IDSCP2 Exception
 *
 * @author Leon Beckmann (leon.beckmann@aisec.fraunhofer.de)
 */
class Idscp2Exception : RuntimeException {
    constructor(message: String?) : super(message) {}
    constructor(message: String?, cause: Throwable?) : super(message, cause) {}
}